<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>83</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Set Voltage</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QLCDNumber" name="count">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="numDigits">
      <number>5</number>
     </property>
     <property name="digitCount">
      <number>5</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>SET</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="voltBox">
     <property name="minimum">
      <number>100</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="voltageReadback">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Frame</receiver>
   <slot>setSpecifiedVoltage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>voltBox</sender>
   <signal>editingFinished()</signal>
   <receiver>Frame</receiver>
   <slot>setSpecifiedVoltage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setVoltage(int)</slot>
  <slot>setWavelength(int)</slot>
  <slot>setSpecifiedVoltage()</slot>
 </slots>
</ui>
